src/main/kotlin/calculator/Splitter.kt

```diff
@@ -0,0 +1,20 @@
+package calculator
+
+class Splitter {
+    companion object {
```

Comment:
```suggestion
object Splitter {
```
클래스를 인스턴스화하지 않아도 된다면, object를 활용해보면 어떨까요?